app/locales/zh-tw/translations.coffee

```diff
@@ -9,7 +9,7 @@ I18nTranslationsZhTw =
     "submitted": "已提交"
     "messages_title" : "訊息"
     "select": "選擇"
-    "QuotaExceededError": "抱歉，此網站未能支援Safari <b>private mode.</b> 請嘗試</br><ul><li><a href='https://itunes.apple.com/in/app/goodcityadmin.hk/id1013288708?mt=8' style='color: black!important; background-color: #dee4eb !important;'>Downloading the iOS App</a></li><li>Using regular (not private) mode in Safari</li><li>Using Chrome's private browsing mode</li></ul>"
+    "QuotaExceededError": "抱歉，此網站未能支援Safari <b> 私人瀏覽模式 </b> 請嘗試</br><ul><li><a href='https://itunes.apple.com/in/app/goodcityadmin.hk/id1013288708?mt=8' style='color: black!important; background-color: #dee4eb !important;'> iOS App 下載中 </a></li><li>正在使用Safari 正常(非無痕) 模式 </li><li> 正在使用Chrome 的私人瀏覽模式 </li></ul>"
```

Comment:
This is done with the translation with missing parts in line 12. 
Should Nokia pull this request?
Thanks.